src/e2e/java/teammates/e2e/pageobjects/InstructorNotificationsPage.java

```diff
@@ -0,0 +1,17 @@
+package teammates.e2e.pageobjects;
+
+/**
+ * Page Object Model for instructor notifications page.
+ */
+public class InstructorNotificationsPage extends UserNotificationsPage {
+
+    public InstructorNotificationsPage(Browser browser) {
+        super(browser);
+    }
+
+    @Override
+    protected boolean containsExpectedPageContents() {
+        return getPageSource().contains("Notifications");
```

Comment:
Should this be "Instructor Notifications"?

Reply:
Thanks for the catch! I've pushed a quick fix